en/docs/ipaas/micro-integrator/develop-integrations-with-integration-studio.md

```diff
@@ -57,9 +57,13 @@ During deployment, Choreo performs a Trivy scan to detect critical vulnerabiliti
 
 ## Environment variables
 
-You can use environment variables in WSO2 Micro Integrator to improve the configuration management, security, portability, and manageability of the Micro Integrator installation. By using environment variables, organizations can simplify the management and maintenance of their micro installations and ensure that they can be quickly and easily updated when required.
+You can use environment variables when developing integration artifacts with WSO2 Integration Studio to improve configuration management, security, portability, and manageability. By using environment variables, organizations can simplify the management and maintenance of their integrations and ensure that they can be quickly and easily updated when required.
 
-The Choreo DevOps Portal allows you to define configurations and secrets to a deployed integration (as well as other component types). For more information, see [Configurations and secrets](../../devops/devops-portal.md#configurations-and-secrets).
+The Choreo Console allows you to define environment variables for your integration project. You can define the environment variables before you deploy the component.
+
+![Environment Variables](../../assets/img/ipaas/integration-studio/env_variables.png){.cInlineImage-half}
+
+The Choreo DevOps Portal also allows you to define configurations and secrets to a deployed integration (as well as other component types) with more flexibility to cater your advance configuration needs. For more information, see [Configurations and secrets](../../devops/devops-portal.md#configurations-and-secrets).
 
 For a list of parameters that you can configure as environment variables, see [WSO2 API Manager Documentation - Injecting Parameters - Supported parameters](https://apim.docs.wso2.com/en/latest/integrate/develop/injecting-parameters/#supported-parameters).
```

Comment:
```suggestion
For a comprehensive list of parameters that can be configured as environment variables, please refer to the [WSO2 API Manager Documentation - Injecting Parameters - Supported parameters](https://apim.docs.wso2.com/en/latest/integrate/develop/injecting-parameters/#supported-parameters).
```

Reply:
Fixed in 0a219c3d8424e846c56f75bacb3c16c854cae75d